data-generator/dataWriter-requests.js

```diff
@@ -0,0 +1,49 @@
+// there is still a problem with this setup
+// but it probably can be ignored
+// requests are ordered, but their times are all random for one day
+// which I guess is fine, but in practice everything will be accessing the most recent
+// still I guess this is okay
+
+var fs = require('fs');
```

Comment:
Mix of var and const declarations, would probably be better to use const when possible to avoid accidental reassignments.